From b1fd9e1d7cf5ec58433a4c73a2b7a9e219bcda34 Mon Sep 17 00:00:00 2001
From: Sheng Nan <b38800@freescale.com>
Date: Mon, 4 Mar 2013 10:35:13 +0800
Subject: [PATCH 123/231] ENGR00252411: mxc_v4l2_capture: ov5642/ov5640 cannot
 both be set as build-in.

ov5642 and ov5640 are inside choice "prompt "Select Camera/TV Decoder""
It prevents some modules are set as build-in at the same time.

All the cameras inside the choice are parallel cameras. There is no need to
include all of them inside a choice.

Know issues:
ADV7180 (out of the choice) and mipi ov camera (inside the choice)
can not be both selected for ARM2 board:
CSI0_MCLK pad is conflicted between adv7180 and mipi camera ext port.
Developers should know this.

Signed-off-by: Sheng Nan <b38800@freescale.com>
---
 drivers/media/video/mxc/capture/Kconfig |    7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/media/video/mxc/capture/Kconfig b/drivers/media/video/mxc/capture/Kconfig
index 3013155..a862636 100644
--- a/drivers/media/video/mxc/capture/Kconfig
+++ b/drivers/media/video/mxc/capture/Kconfig
@@ -26,11 +26,6 @@ config VIDEO_MXC_CSI_DMA
 	  with this method the CSC (Color Space Conversion) and resize are not supported.
 	  If unsure, say N.
 
-choice
-	prompt "Select Camera/TV Decoder"
-	default MXC_CAMERA_OV3640
-	depends on VIDEO_MXC_CAMERA
-
 config MXC_CAMERA_MC521DA
 	tristate "Magnachip mc521da camera support"
 	select I2C_MXC
@@ -99,8 +94,6 @@ config MXC_CAMERA_OV5642
 	---help---
 	  If you plan to use the ov5642 Camera with your MXC system, say Y here.
 
-endchoice
-
 config MXC_TVIN_ADV7180
 	tristate "Analog Device adv7180 TV Decoder Input support"
 	depends on (MACH_MX35_3DS || MACH_MX51_3DS || MACH_MX6Q_SABREAUTO || MACH_MX6Q_ARM2)
-- 
1.7.10.4

